deploy/setup/package.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "wbs-deploy-setup",
+	"version": "1.0.0",
```

Comment:
This is not integrated in nx, because it is not a separate product I suppose? Do we really track a version for that or can we maybe even just skip this field and consider this package part of deploy?

Reply:
For now this is `private: true` so not managed by nx. But for several reasons I do think this packaged would be versioned separate from the Deploy package. As highly related products they technically actually have very different concerns and are minimally dependent on each other. In the end deploy/setup's interface with that product ends with the generation of a `.env` file and the first run of a `docker compose up -d`.

Reply:
I get the theoretical idea. I still do not see what this version communicates to who and when it is updated by who based on which rules.

What I want to prevent is a semi-hidden version number which looks like communicating something, but in the end is just never updated.

Reply:
I can see all your points. Still, leaving a `1.0.0` here without a way set how to continue does not feel good to me. Especially because all the other versions in this repo have their clear definitions and processes.

What about setting the version to 0.0.0 for now? It at least declares the version kind of obviously not handled yet, and move on to higher numbers as soon as the NX integration is there. Alternatively, I would be also up for adding the NX integration now.

I am not a fan of the current state "in between".